Raven.Database/Storage/Voron/Schema/Updates/From13To14.cs

```diff
@@ -0,0 +1,138 @@
+    {
+        public override string FromSchemaVersion => "1.3";
+
+        public override string ToSchemaVersion => "1.4";
+
+        public override void Update(TableStorage tableStorage, Action<string> output)
+        {
+            foreach (var multiTreeName in new []
+            {
+                tableStorage.DocumentReferences.GetIndexKey(Tables.DocumentReferences.Indices.ByKey),
+                tableStorage.DocumentReferences.GetIndexKey(Tables.DocumentReferences.Indices.ByRef),
+                tableStorage.DocumentReferences.GetIndexKey(Tables.DocumentReferences.Indices.ByView),
+                tableStorage.DocumentReferences.GetIndexKey(Tables.DocumentReferences.Indices.ByViewAndKey),
+
+                tableStorage.Lists.GetIndexKey(Tables.Lists.Indices.ByName),
+
+                tableStorage.MappedResults.GetIndexKey(Tables.MappedResults.Indices.ByViewAndDocumentId),
+                tableStorage.MappedResults.GetIndexKey(Tables.MappedResults.Indices.ByView),
+                tableStorage.MappedResults.GetIndexKey(Tables.MappedResults.Indices.ByViewAndReduceKey),
+                tableStorage.MappedResults.GetIndexKey(Tables.MappedResults.Indices.ByViewAndReduceKeyAndSourceBucket),
+
+                tableStorage.ReduceKeyCounts.GetIndexKey(Tables.ReduceKeyCounts.Indices.ByView),
+
+                tableStorage.ScheduledReductions.GetIndexKey(Tables.ScheduledReductions.Indices.ByView),
+                tableStorage.ScheduledReductions.GetIndexKey(Tables.ScheduledReductions.Indices.ByViewAndLevelAndReduceKey),
+
+                tableStorage.ReduceResults.GetIndexKey(Tables.ReduceResults.Indices.ByViewAndReduceKeyAndLevelAndSourceBucket),
+                tableStorage.ReduceResults.GetIndexKey(Tables.ReduceResults.Indices.ByViewAndReduceKeyAndLevel),
+                tableStorage.ReduceResults.GetIndexKey(Tables.ReduceResults.Indices.ByViewAndReduceKeyAndLevelAndBucket),
+                tableStorage.ReduceResults.GetIndexKey(Tables.ReduceResults.Indices.ByView),
+
+                tableStorage.Queues.GetIndexKey(Tables.Queues.Indices.ByName),
+
+                tableStorage.Tasks.GetIndexKey(Tables.Tasks.Indices.ByType),
+                tableStorage.Tasks.GetIndexKey(Tables.Tasks.Indices.ByIndex),
+                tableStorage.Tasks.GetIndexKey(Tables.Tasks.Indices.ByIndexAndType)
+            })
+            {
+                using (var tx = tableStorage.Environment.NewTransaction(TransactionFlags.ReadWrite))
+                {
+                    var table = tx.ReadTree(multiTreeName);
+
+                    if (table.State.EntriesCount == 0)
+                    {
+                        output($"Multi tree {multiTreeName} is empty. No need to migrate.");
+                        continue;
+                    }
+
+                    output($"Migrating {multiTreeName}, with {table.State.EntriesCount:#,#;;0} entries");
+
+                    tableStorage.Environment.CreateTree(tx, "temp_" + multiTreeName);
```

Comment:
I don't think we need to remove it. In case of a failed previous migration attempt we would just overwrite old records.